app/services/automation_rules/action_service.rb

```diff
@@ -1,53 +1,90 @@
 class AutomationRules::ActionService
-  def initialize(rule, conversation)
+  def initialize(rule, account, conversation_ids)
```

Comment:
conversation_ids could be confusing as it gives an impression that we are passing an array of conversation_ids. But from my understanding, we are only passing a single conversation_id?

Reply:
No, it changes, could be conversation_ids or single, but we do send an array for some actions.